pr_agent/algo/utils.py

```diff
@@ -355,7 +357,16 @@
         except:
             pass
 
-    # thrid fallback - try to remove last lines
+     # third fallback - try to remove leading and trailing curly brackets
+    response_text_copy = response_text.strip().rstrip().removeprefix('{').removesuffix('}')
+    try:
+        data = yaml.safe_load(response_text_copy,)
+        get_logger().info(f"Successfully parsed AI prediction after removing curly brackets")
+        return data
+    except:
+        pass
```

Comment:
**Suggestion:** In the `try_fix_yaml` function, the `except` clause is empty. This can make debugging difficult because it swallows all exceptions. Consider logging the exception or re-raising it after handling. [best practice]